GyroData/Scenes/MotionMeasurementScene/MeasurementViewController.swift

```diff
@@ -110,13 +143,50 @@ extension MeasurementViewController {
         stopButton.addTarget(self,
                              action: #selector(stopButtonTapped),
                              for: .touchUpInside)
+        
+        segmentedControl.addTarget(self,
+                                   action: #selector(segmentedContorolSelected),
+                                   for: .valueChanged)
     }
     
     @objc private func measureButtonTapped() {
-        MotionMeasurementManager.shared.startMeasurement(MotionType.acc ,on: graphView)
+        measurementviewModel.startMeasurement(motionType, on: graphView)
     }
     
     @objc private func stopButtonTapped() {
-        MotionMeasurementManager.shared.stopMeasurement(MotionType.acc)
+        measurementviewModel.stopMeasurement(motionType)
+    }
+    
+    @objc private func segmentedContorolSelected() {
+        switch segmentedControl.selectedSegmentIndex {
+        case 0:
+            motionType = .acc
+        case 1:
+            motionType = .gyro
+        default:
+            motionType = .acc
+        }
+    }
+}
+
+extension MeasurementViewController {
+    private func showAlert(message: String) {
+        let alert = UIAlertController(title: "알림",
+                                      message: message,
+                                      preferredStyle: .alert)
+        
+        alert.addAction(UIAlertAction(title: "확인", style: .default))
+        self.present(alert, animated: true)
+    }
+    
+    private func showLoading() {
+        indicatorView.frame = view.frame
+        indicatorView.color = .systemGray
+        view.addSubview(indicatorView)
+        indicatorView.startAnimating()
+    }
+    
+    private func hideLoading() {
+        indicatorView.removeFromSuperview()
```

Comment:
```suggestion
         DispatchQueue.main.asyncAfter(deadline: .now() + .seconds(1)) {
            self.indicatorView.stopAnimating()
        }
```
stopAnimating을 하면 될것같구요!

주디가 남겨주신 멘트에 indicator가 안보인다고 하셨는데 DispatchQueue를 이용해서 의도적으로 지연줄 수 있을 것 같습니다!

Reply:
다시 보니 저장 버튼 action에 화면을 나가는 코드가 있어서 바로 나가버려서 볼 수가 없는 문제도 있었네요 😅
나가는 코드를 `hideLoading`으로 옮겼는데도 너무 빨라서 위에 써주신 코드도 같이 넣었습니다!